guides/common/modules/con_adding-a-red-hat-subscription-to-your-project-server.adoc

```diff
@@ -0,0 +1,14 @@
+[id="adding-a-red-hat-subscription-to-your-project-server-{context}"]
+= Adding a Red{nbsp}Hat subscription to your {ProjectServer}
```

Comment:
I am unsure about "adding" and "{ProjectServer}". The concept is about importing a RH manifest to {Project}, not adding a valid RHEL subscription to your Foreman Server.

With ACS, one could IMHO argue that even Smart Proxy Servers access RH CDNs to get packages.

Reply:
Addressed (including renaming the file etc.)